File: src/pages/show/utils/NumUtils.js
/**
 * @author xzm
 * @date 2019/5/14
 */

const formatShowNum = (num) => {
    if (num <= 999) {
        return num + "";
    }

    if (num > 999 && num <= 100000) {
        return "999+";
    }

    return "10w+";
};

export default {
    formatShowNum
}

File: android/app/src/main/java/com/meeruu/sharegoods/rn/showground/utils/NumUtils.java
package com.meeruu.sharegoods.rn.showground.utils;

public class NumUtils {
    public static String formatShowNum(int num) {
        if (num <= 999) {
            return num + "";
        } else if (num < 10000) {
            return num / 1000 + "K+";
        } else if (num < 100000) {
            return num / 10000 + "W+";
        } else {
            return "10w+";
        }
    }
}
